3-UsingAPIs/Week1/homework/ex4-pokerDiceAll.js

```diff
@@ -20,13 +20,11 @@ function rollTheDices() {
   const dices = [1, 2, 3, 4, 5];
   const arrayOfPromises = dices.map(el => rollDice(el))
 
-  Promise.all(arrayOfPromises).then(values => {
-    return Promise.all(values.map(r => r
-    )).then((results) => console.log('Resolved!', results))
-  }).catch((error) => console.log('Rejected!', error.message));
+  return Promise.all(arrayOfPromises).then(values => values)
```

Comment:
This `.then()` does add nothing. It just returns the same values that `Promise.all(arrayOfPromises)` already returns.

Apart from, this is the solution I was looking for.